crypto/sm2/sm2_crypt.c

```diff
@@ -362,6 +380,11 @@ int ossl_sm2_decrypt(const EC_KEY *key,
         goto done;
     }
 
+    if (is_all_zeros(msg_mask, msg_len)) {
+        ERR_raise(ERR_LIB_SM2, ERR_R_INTERNAL_ERROR);
```

Comment:
Could you use SM2_R_INVALID_ENCODING instead of ERR_R_INTERNAL_ERROR?

Reply:
OK, I'll fix this.